framework/src/auxkernels/FunctorElementalGradientAux.C

```diff
@@ -25,6 +25,19 @@ FunctorElementalGradientAuxTempl<is_ad>::validParams()
   params.addParam<MooseFunctorName>("factor", 1, "A factor to apply on the functor");
   params.addParam<MaterialPropertyName>(
       "factor_matprop", 1, "A (regular) material property factor to apply on the functor");
+
+  // We need some ghosting for the Finite Volume Fields (we use neighbor information to compute
+  // gradient)
+  params.addParam<unsigned short>("ghost_layers", 1, "The number of layers of elements to ghost.");
+  params.addRelationshipManager(
+      "ElementSideNeighborLayers",
+      Moose::RelationshipManagerType::GEOMETRIC | Moose::RelationshipManagerType::ALGEBRAIC |
+          Moose::RelationshipManagerType::COUPLING,
```

Comment:
Oh you should remove coupling from this list as this is not a residual/Jacobian object

Reply:
Thank Alex again! I'll definitely start looking at the relationship manager functions after this, lot to learn!